.gitignore

```diff
@@ -1 +1,13 @@
 /fev_prove
+SymbiYosys/
+yosys/
+fev
+venv
+.env*
+tmp
+history
+current
+messages*
+current
```

Comment:
SymbiYosys and yosys installs should be outside of the repo dir. I was planning to capture conversion histories as world-visible training data, but I think it would be better to do conversions in other repos/directories outside of this repo. Duplicate entry for `current`. Fine for now, but food for thought.

Reply:
Even when converting in this directory, ignore the whole conversion directory vs. individual files/file-types.